LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 The Warp Team
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
```

Comment:
Thank you for flagging and working on this! Just so we're consistent with other repos (https://github.com/warpdotdev/keysets, https://github.com/warpdotdev/themes), would you mind making this the apache license instead?

Reply:
I think we can close this one out since https://github.com/warpdotdev/workflows/pull/71 was merged